backend/src/layer/data/swka_parser/swka_resolver.rs

```diff
@@ -0,0 +1,88 @@
+use crate::interface::mensa_parser::ParseError;
+use futures::future::join_all;
+use reqwest::Client;
+use std::time::Duration;
+use tracing::log::debug;
+
+pub struct SwKaResolver;
+
+impl SwKaResolver {
+    pub const fn new() -> Self {
+        Self
+    }
+
+    fn get_client() -> Result<Client, ParseError> {
+        let client = Client::builder() // dont change or tests will fail!
+            .danger_accept_invalid_certs(true) //probably not needed
+            .timeout(Duration::from_millis(1000))
+            .connect_timeout(Duration::from_millis(1000))
+            .user_agent("User-Agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.114 Safari/537.36")
```

Comment:
we should probably provide our own user agent string:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent
https://docs.rs/reqwest/latest/reqwest/struct.ClientBuilder.html#method.user_agent